.claude/skills/build/SKILL.md

```
@@ -0,0 +1,30 @@
+## Sub-agent prompt
+
+Use the Agent tool with a timeout of 180000ms:
+
+**Command logic:**
+- No arguments or `core` -> `npm run build -w packages/core`
+- `html` or `storybook` -> `npm run build -w packages/html`
+- `examples` -> `./scripts/build-all-examples.bash`
+- `all` -> `npm run build -w packages/core && ./scripts/build-all-examples.bash && npm run build -w packages/html`
+- Other -> `npm run build -w packages/$ARGUMENTS`
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)